### DISCUSSION TITLE
Each scenario run two times

### DISCUSSION THREAD

**Discussion by SurglogsGithubUser** [Tyler-Keith-Thompson/CucumberSwift#85] [Q&A]
Is there any reason, why should my scenarios run two times? I cannot find any settings for this. They even run two times when they are successful.

**Comment by Tyler-Keith-Thompson** [comment]
So Xcode has some weirdness that caused the lib to change how it finds tests to run. Are you using the latest from here: https://tyler-keith-thompson.github.io/CucumberSwift/tutorials/cucumberswift/spm-step-by-step where you I've it something that inherits from CucumberTest with the latest version of the library?

**Reply by SurglogsGithubUser** [reply]
I use CocoaPods, should rather I use it trough SPM?

**Comment by MarekSlaninka** [comment]
Hi, did You found, why is this happening? Thanks

**Comment by Tyler-Keith-Thompson** [comment]
Apologies, I haven't had time to look into this yet. What I am going to do is turn it into an issue, though. It'll provide higher visibility to others who may submit a PR and it'll get prioritized more appropriately by me when I do have time.

**Comment by Tyler-Keith-Thompson** [comment]
I'm not able to repro this, but we recently changed how the output looks in the test navigator and I think maybe it's just causing confusion. Because the scenario shows up once per step. Is this what you're seeing?

<img width="452" alt="Screenshot 2023-04-29 at 4 29 16 PM" src="https://user-images.githubusercontent.com/33705774/235326683-4742b090-8e6a-4211-b460-3697a02b4de7.png">

I feel like every time we try and make the test report nicer Xcode pulls the rug out from under us.
